lib/model/store.dart

```diff
@@ -215,6 +219,17 @@ class PerAccountStore extends ChangeNotifier {
     } else if (event is AlertWordsEvent) {
       assert(debugLog("server event: alert_words"));
       // We don't yet store this data, so there's nothing to update.
+    } else if (event is UserSettingsUpdateEvent) {
+      assert(debugLog("server event: user_settings/update"));
+      switch (event) {
+        case UserSettingsUpdateEventTwentyFourHourTime():
+          userSettings!.twentyFourHourTime = event.value;
```

Comment:
I see, OK.

Still, ideally I'd like to avoid crashing even if a server behaves oddly. Writing `?.` is as easy as `!.`, so we might as well.

(If we wanted to be paranoid we could plan to log if we get one of these events and `userSettings` is null. But that seems like an unlikely server bug, and when we start requiring Server 5 it'll become impossible because `userSettings` will be non-nullable (so if the server did leave it out, we'd give an error on trying to connect to it).